data-modeling.md

```diff
@@ -0,0 +1,118 @@
+
+![Connecting Nodes](imgs/data-modeling-connecting-nodes.png)
+
+
+Adding multiple nodes and creating a network of connections between them defines how input data should be transformed by the data flow graph to create the derived output data.
+
+## Nodes:
+
+
+![Node List](imgs/data-modeling-node-list.png)
+
+
+### Load Node
+
+![Load Node](imgs/data-modeling-load-node.png)
+
+
+The Load Node is how data is brought into the data modeling process. Each Load Node represents **one** feature from a dataset. You select this feature with the `Data Source` dropdown at the top of the node. You'll see features listed per dataset in this dropdown.
+
+The following options in the Load Node relate to how data regridding is handled.
+
+The first of these is a global target resolution that applies to all your features. You need to select both a geo and temporal target resolution. You can either manually hardcode these (see [Manual Resolution](#manual-resolution) below), or you can check one or both of the boxes on a Load Node to make its resolution be the target resolution. For the entire graph, you can only have one geo and one temporal target resolution, either in the Load Nodes or in the manual selectors in the sidebar.
+
+The second is selecting regridding methods that will be used if the data in the load node needs to be regridded. These are specified with the Geo and Time Aggregation function dropdowns for each Load Node:
+- `conserve` - maintain the total sum of the data before and after (e.g. for regridding population)
+- `min` - take the minimum value from each bucket during regridding
+- `max` - take the maximum value from each bucket during regridding
+- `mean` - take the mean value from each bucket during regridding. *Note: Use interp_or_mean instead of `mean`, since it doesn't handle well when increasing the resolution of the data*
+- `median` - take the median value from each bucket during regridding
+- `mode`  - take the mode of each bucket during regridding. *Note: Use nearest_or_mode instead of `mode`, since it doesn't handle well when increasing the resolution of the data*
+- `interp_or_mean`  - if increasing resolution, interpolate the data. If decreasing resolution, take the mean of each bucket
+- `nearest_or_mode`  - if increasing resolution, take the nearest data. if decreasing resolution, take the mode of each bucket
+
+### Threshold Node
+![Threshold Node](imgs/data-modeling-threshold-node.png)
+
+
+Threshold Nodes are used to select specific portions of data where some condition is true.
+
+Take the example of a temperature dataset where you want to filter for values above a threshold temperature. A Threshold Node that takes temperature data as input and is set to `greater_than` with the value of your threshold creates a boolean mask that is true wherever the condition was met. This can then be used to filter the original data with a [Join Node](#join-node).
```

Comment:
I changed `A filter node that takes temperature data as input...` to `A Threshold Node that takes temperature data as input...` because I think using a different name for the node here is confusing.